src/passwordValidation.js

```diff
@@ -0,0 +1,58 @@
+export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
+
+/**
+ * Checks if a given password is valid or invalid.
+ * If valid it returns true, otherwise false
+ * @param {string} password
+ * @returns {boolean}
+ */
+
+export default function isValidPassword(password = "") {
+  // The following line ensures, that password is always a string, like the number 128 -> string "128"
+  if (typeof password !== "string") password = String(password);
+
+  if (password.length !== 10) {
```

Comment:
There is no need to add curly braces here; it can be written in one line of code.